dataland-frontend/tests/component/components/forms/parts/elements/derived/ToggleKpisDataPoint.cy.ts

```diff
@@ -21,6 +21,8 @@ describe("Component tests for toggle data point", () => {
           qualityValueBeforeDataPointWasDisabled: "",
         };
       },
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```

Comment:
There are ways around this. You could use setProps as done in other tests

Reply:
setProps didn't work for me, for some reason it didn't see the set variables and props worked, so I allowed myself to use this method